src/hooks/useHandleSelectorComponent.tsx

```diff
@@ -21,7 +29,7 @@ export const useHandleSelectorComponent = ({ item, type }: { item: unknown; type
       store.dispatch(setCreateTemplateFields({ targetField: 'workflowStateId', value: (item as WorkflowStateResponse)?.id }))
     }
 
-    if (type === SelectorType.ASSIGNEE_SELECTOR && item) {
+    if ((mode === handleSelectorComponentModes.CreateTaskFieldUpdate, type === SelectorType.ASSIGNEE_SELECTOR) && item) {
```

Comment:
`mode === handleSelectorComponentModes.CreateTaskFieldUpdate,` 

What's up with the comma here? I'm not sure if this is valid JS...